lib/except.js

```diff
@@ -21,12 +21,18 @@ function makeExceptionMaker(type)
     if (arguments.length > 1) {
       message = util.format.apply(null, [message].concat(Array.prototype.slice.call(arguments, 1)));
     }
-    var retVal = new Error(message);
-    retVal.type = type;
-    return retVal;
+    return new ExceptionMakerError(message, type);
   };
 }
 
+// Create a new object, that prototypally inherits from the Error constructor
+function ExceptionMakerError(message, type) {
+  this.type = type;
```

Comment:
Okay, really minor nit, but I would initialize the values in the same order as the function signature.  So, message then type.  Otherwise, LGTM.